uisdk/src/main/java/tech/dojo/pay/uisdk/presentation/components/BasicCardNumberInPutField.kt

```diff
@@ -178,7 +183,7 @@ internal fun BasicCardNumberInputField(
                     .fillMaxWidth()
                     .run { if (focusRequester != null) focusRequester(focusRequester) else this }
             )
-            if (cardNumberValue.text.isNotBlank()) {
+            if (cardNumberValue.text.isNotBlank() && CardCheckoutScreenValidator().isCardSchemaSupported(cardNumberValue.text, supportedCardSchemas)) {
```

Comment:
just a QQ why do we need to have the `CardCheckoutScreenValidator`. here as we already did that on the viewModel to check if the card Valid or not ?

Reply:
This one is different. Here we don't validate a card yet but we're showing its card schema icon. We don't want to show icons for schemas that are not supported as user is typing.